tdrs-frontend/src/components/Home/Home.jsx

```diff
@@ -194,19 +195,60 @@ function Home() {
   }
 
   return (
-    <div className="margin-top-5">
-      <p className="margin-top-1 margin-bottom-4">
-        You've been approved as a(n) {role.name}. You'll be able to do the
-        following in TDP:
-      </p>
-      <ul>
-        {role.permissions?.map((permission) => (
-          <li key={permission.id} id={permission.id}>
-            {permission.name}
-          </li>
-        ))}
-      </ul>
-    </div>
+    <>
+      <div className="margin-top-5">
+        <p className="margin-top-1 margin-bottom-4">
+          You've been approved as a(n) {role.name}. You'll be able to do the
+          following in TDP:
+        </p>
+        <ul>
+          {role.permissions?.map((permission) => (
+            <li key={permission.id} id={permission.id}>
+              {permission.name}
+            </li>
+          ))}
+        </ul>
+        <p>
+          <b> Data Reporting Deadlines </b>
+        </p>
+        {/* uswds striped table 3 x 5 */}
+        <table className="usa-table usa-table--striped">
+          <thead>
+            <tr>
+              <th scope="col">Fiscal Quarter</th>
+              <th scope="col">Calander Period</th>
+              <th scope="col">Due Date</th>
+            </tr>
+          </thead>
+          <tbody>
+            <tr>
+              <td>1</td>
+              <td>Oct 1 - Dec 31</td>
+              <td>February 14</td>
+            </tr>
+            <tr>
+              <td>2</td>
+              <td>Jan 1 - Mar 31</td>
+              <td>May 15</td>
+            </tr>
+            <tr>
+              <td>3</td>
+              <td>Apr 1 - Jun 30</td>
+              <td>August 15</td>
+            </tr>
+            <tr>
+              <td>4</td>
+              <td>Jul 1 - Sep 30</td>
+              <td>November 14</td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+      <br />
+      <hr />
+      <br />
+      <ResourceCards />
```

Comment:
could `ResourceCards` potentially be made into a more generic component? implementation in this component might look like this

```
<ResourceCards
    cards={[
        { title: 'TDP Knowledge Center', description: 'The knowledge center contains resources on all things TDP from account creation to data submission.' },
        { title: 'Transmission File Layouts & Edits', description: 'All transmission file layouts and edits (i.e. error codes) for TANF and SSP-MOE data reporting.' },
        // ... etc.
    ]}
/>
```